test/e2e/deployment_appwrapper_test.go

```diff
@@ -0,0 +1,175 @@
+/*
+Copyright 2024.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+	mcadv1beta2 "github.com/project-codeflare/appwrapper/api/v1beta2"
+	. "github.com/project-codeflare/codeflare-common/support"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/utils/ptr"
+	"sigs.k8s.io/kueue/apis/kueue/v1beta1"
+)
+
+func TestDeploymentAppWrapperCpu(t *testing.T) {
+	runDeploymentAppWrapper(t)
+}
+
+func TestDeploymentAppWrapperGpu(t *testing.T) {
+	runDeploymentAppWrapper(t)
+}
```

Comment:
Just a note, in other tests the CPU/GPU test split is done as GPU test has adjusted configuration.
If you don't have any specific GPU related code feel free to keep only one test function.

Reply:
Thanks.  Done.